buildscripts/buildvars-setup.cmd

```diff
@@ -104,6 +106,7 @@ for /f "delims=" %%a in ('powershell -NoProfile -ExecutionPolicy ByPass "& ""%__
 
 set __VSProductVersion=
 if /i "%__VSVersion%" == "vs2015" set __VSProductVersion=140
+if /i "%__VSVersion%" == "vs2017" set __VSProductVersion=150
```

Comment:
And update https://github.com/dotnet/corert/blob/master/Documentation/prerequisites-for-building.md too